requirements.txt

```diff
@@ -2,3 +2,4 @@ beautifulsoup4
 numpy
 selenium
 requests
+wikipediaapi
```

Comment:
This is different from the package in `setup.py`, which one is it?

Reply:
Yes, that seems the right one, since that is what you need for installing it through PyPi.